tests/base_gmsh/Makefile

```
@@ -0,0 +1,17 @@
+all: params.log
+
+.PHONY: all base_case.py
```

```suggestion
.PHONY: all
```

Also I guess we should make `clean` a phony target throughout everything...?

---

Yeah, I think so, strictly speaking. I guess we can start doing that here and deal with the rest later.